src/component/mxgraph/BpmnCellRenderer.ts

```diff
@@ -26,12 +26,12 @@ export class BpmnCellRenderer extends mxgraph.mxCellRenderer {
     const overlays = graph.getCellOverlays(state.cell);
     let dict = null;
 
-    if (overlays != null) {
+    if (overlays != undefined) {
```

Comment:
**nitpick:** restore null. This is mxGraph code and the rule is disabled.
This applies everywhere in the file